docs/earthly-config/earthly-config.md

```diff
@@ -163,6 +163,10 @@ Allows overriding Earthly's automatic MTU detection. This is used when configuri
 
 Allows overriding Earthly's automatic `ip_tables` module detection. Valid choices are `iptables-legacy` or `iptables-nft`.
 
+### git_image
```

Comment:
thanks for adding docs too!

our docs are a bit complicated because anything that gets merged to `main` will be automatically deployed, which might confuse users since the docs will reference config options that have not been released in the latest version.

can you move this to a separate PR that's opened against the `next` branch instead of `main`? That way when we perform the next release, we'll merge `next` into `main` to keep docs in sync.